docs/Getting-Started.md

```diff
@@ -0,0 +1,270 @@
+      },
+
+      handleRemove: function (e) {
+        e.preventDefault();
+        this.nearestOwnerProps.remove(this.props.todo);
+      },
+
+      render: function () {
+        return (
+          <div>
+            <p>{ this.props.todo.text }</p>
+            <button onClick={this.handleRemove}>Delete</button>
+          </div>
+        );
+      }
+    });
+
+    module.exports = Todo;
+
+```
+
+## TodoCreateForm.jsx
+Another of the opinionated `Tux` components are the Ownee class. They receive their dynamic props through standard `this.props` sharing, and unless it is needed for input form validation. They don't need to manage their own state standardly as they receive their static methods and props from their closest Owner component. The `TodoCreateForm` utilizes this by accessing the `add` method from its nearest owner component, which is the `TodoViewOwner`.
+
+```javascript
+    var React = require('tux/React');
+
+    var TodoCreateForm = React.createOwneeClass({
+      // nearestOwnerPropTypes allows us to perform prop validation on our nearestOwnerProps
+      nearestOwnerPropTypes: {
+        add: React.PropTypes.func.isRequired
+      },
+
+      handleSubmit: function (e) {
+        e.preventDefault();
+        var todoTextInput = this.refs.textInput.getDOMNode();
+
+        this.nearestOwnerProps.add({
+          text: todoTextInput.value
+        });
+
+        todoTextInput.value = '';
+      },
+
+      render: function () {
+        return (
+          <form onSubmit={this.handleSubmit}>
+            <input type="text" ref="textInput" placeholder="Add a Todo" />
+            <button type="submit">Add</button>
+          </form>
+        );
+      }
+    });
+
+    module.exports = TodoCreateForm;
+```
+
+## todoActions.js
+`Tux` leverages the `Flux` architecture but abstracts away all of the normal boilerplate associated with `Flux`. In `Tux` we build `actionCategories` and then we can invoke them in order to dispatch their actions and register with them in order to receive these actions. Here is the `todoActions` category that we required in from before.
+
+```javascript
+    var Actions = require('tux/Actions');
+
+    var todoActions = Actions.createActionCategory({
+      category: 'todos',
+      source: 'todo views',
+      actions: ['add', 'remove'] // the specified actions will become methods on the todoActions ActionCategory. These actions allow us to get data from our views to our actions and then to our stores
+    });
+
+    module.exports = todoActions;
+```
+
+## todoStore.js
+`Tux` provides all of the glue code needed to build stores and register them with the `TuxActions` dispatcher. Now that we have created our actions and views. Let's create our store for our Todos that respond to our actions and provide the data for our views. `Tux` here again significantly reduces the boilerplate required to get a store up and running.
+
+```javascript
+    var ActionStores = require('tux/Stores/ActionStores');
```

Comment:
Add line to require in the `todoActions` and explain we are doing this to make sure the actions have been created before we attempt to register with them.